### ISSUE TITLE
Provide more descriptive error messages

### ISSUE THREAD

**Issue by ericksoen** [auth0/auth0-deploy-cli#227]
**Please do not report security vulnerabilities here**. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.

**Thank you in advance for helping us to improve this library!** Your attention to detail here is greatly appreciated and will help us respond as quickly as possible. For general support or usage questions, use the [Auth0 Community](https://community.auth0.com/) or [Auth0 Support](https://support.auth0.com/). Finally, to avoid duplicates, please search existing Issues before submitting one here.

By submitting an Issue to this repository, you agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Describe the problem you'd like to have solved

I'm a newish user to the tool (a good thing!) using a YAML configuration and ran into an issue where I had rules defined in my YAML configuration that were not in the `\rules` directory. When I ran the `aodeploy` tool, I received the following error message:

```bash
$ a0deploy import --config_file config.json -i tenant.yaml
2020-04-21T19:18:19.032Z - error: Problem running command import
2020-04-21T19:18:19.035Z - error: Problem deploying rules
```

This didn't give me a lot of clues regarding any potential misconfiguration issues. I'm fairly good at debugging open source tools so I was able to find where the [error message originated](https://github.com/auth0/auth0-deploy-cli/blob/f6b613bdac8ce03716d82b0597d76a0c76705526/src/context/yaml/index.js#L72-#L73) and identify that there is a `log.debug` message in the same code block that prints out the error stack.

However, in order to access these debug logs (there may be a faster way since I'm newish to Node development), I had to modify the [Winston log level to debug](https://github.com/auth0/auth0-deploy-cli/blob/f6b613bdac8ce03716d82b0597d76a0c76705526/src/logger.js#L9), build the `auth0-deploy-cli` locally, and then require from the local path. 

When I re-ran a deployment a second time, I receive a more verbose set of error messages:

```bash
2020-04-21T19:09:29.380Z - debug: Loading YAML from \path\to\tenant.yaml
2020-04-21T19:09:29.805Z - debug: Error: Unable to load file \path\to\rules\rule-name.js due to Error: ENOENT: no such file or directory, access '\path\to\rules\rule-name.js'
```

### Describe the ideal solution

Permit configuration of more verbose logging using environment variables. I've used this model previously to debug deployment issues using [Terraform](https://www.terraform.io/docs/internals/debugging.html) and found it to be a highly useful pattern. 

## Alternatives and current work-arounds

> See notes above regarding building deploy CLI locally and requiring.

### Additional context

> Add any other context or screenshots about the feature request here.

**Comment by willvedd** [comment]
@ericksoen Do you feel as though this ticket can be closed with the addition of #228?

**Comment by ericksoen** [comment]
@willvedd Yes! Somehow I missed that that PR was merged. Thank you so much!
